src/App.js

```diff
@@ -234,291 +235,300 @@ const App = () => {
-                onClick={async () => {
-                  const ready = await readyToTransact()
-                  if (!ready) return
-                  sendInternalTransaction()
+            <div className="container notify">
+              <h2>Transaction Notifications with Notify</h2>
+              <div
+                style={{
+                  display: 'flex',
+                  flexDirection: 'column',
+                  alignItems: 'flex-start',
+                  marginBottom: '1rem'
                 }}
               >
-                Send
-              </button>
-              via a internal transaction
+                <div style={{ marginBottom: '1rem' }}>
+                  <label>Send 0.001 Rinkeby Eth to:</label>
+                  <input
+                    type="text"
+                    style={{
+                      padding: '0.5rem',
+                      border: 'none',
+                      borderRadius: '10px',
+                      marginLeft: '0.5rem',
+                      width: '18rem'
+                    }}
+                    value={toAddress}
+                    placeholder="address"
+                    onChange={e => setToAddress(e.target.value)}
+                  />
+                </div>
+                <div>
+                  <button
+                    className="bn-demo-button"
+                    onClick={async () => {
+                      const ready = await readyToTransact()
+                      if (!ready) return
+                      sendHash()
+                    }}
+                  >
+                    Send
+                  </button>
+                  with in-flight notifications
+                </div>
+                <div>
+                  <button
+                    className="bn-demo-button"
+                    onClick={async () => {
+                      const ready = await readyToTransact()
+                      if (!ready) return
+                      sendTransaction()
+                    }}
+                  >
+                    Send
+                  </button>
+                  with pre-flight and in-flight notifications
+                </div>
+                <div>
+                  <button
+                    className="bn-demo-button"
+                    onClick={async () => {
+                      const ready = await readyToTransact()
+                      if (!ready) return
+                      sendInternalTransaction()
+                    }}
+                  >
+                    Send
+                  </button>
+                  via a internal transaction
+                </div>
+              </div>
+              <div>
+                <button
+                  className="bn-demo-button"
+                  onClick={async () => {
+                    if (!address) {
+                      await readyToTransact()
+                    }
+
+                    address && notify.account(address)
```

Comment:
Would address always be defined in this case? so there may not be a need for both bool checks but not sure 🤷🏾

Reply:
I tried to scope this PR to just style changes and nothing functional yet…It comes up in the diff because I wrapped it in a div but ill run the the logic code next week and leave that check as is for now if youre coo with it